app/src/main/res/layout/view_calendar_event_banner.xml

```diff
@@ -35,49 +35,60 @@
             style="@style/BodyMedium"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/marginStandardMedium"
             android:background="@color/calendarHeaderBackground"
             android:drawablePadding="@dimen/marginStandardSmall"
+            android:ellipsize="end"
             android:gravity="center_vertical"
+            android:lines="1"
             android:paddingHorizontal="@dimen/marginStandardMedium"
             android:paddingVertical="@dimen/marginStandardSmall"
             app:drawableStartCompat="@drawable/ic_calendar_infomaniak"
             tools:text="Réunion Produit" />
 
-        <TextView
-            android:id="@+id/pastEventWarning"
-            style="@style/Body.Warning"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginVertical="@dimen/marginStandardMedium"
-            android:layout_marginStart="@dimen/marginStandardMedium"
-            android:drawablePadding="@dimen/marginStandardMedium"
-            android:gravity="center_vertical"
-            android:text="@string/warningEventHasPassed"
-            app:drawableStartCompat="@drawable/ic_warning_big" />
+        <FrameLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
 
-        <TextView
-            android:id="@+id/eventDate"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="@dimen/marginStandardMedium"
-            android:layout_marginBottom="@dimen/marginStandardMedium"
-            android:drawablePadding="@dimen/marginStandardMedium"
-            android:gravity="center_vertical"
-            app:drawableStartCompat="@drawable/ic_calendar"
-            app:drawableTint="@color/iconColor"
-            tools:text="Mardi 28 novembre 2023" />
+            <TextView
+                android:id="@+id/pastEventWarning"
+                style="@style/Body.Warning"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/marginStandardMedium"
+                android:layout_marginBottom="@dimen/marginStandardMedium"
+                android:drawablePadding="@dimen/marginStandardMedium"
+                android:gravity="center_vertical"
+                android:text="@string/warningEventHasPassed"
+                app:drawableStartCompat="@drawable/ic_warning_big"
+                tools:visibility="gone" />
```

Comment:
shouldn't it be the visibility of the views that is gone, and one of them has tools:visibility = visible ? 
It would make more sense